src/plugins/service-wispstock/views/betterportal/products/materio1.tsx

```diff
@@ -1,17 +1,113 @@
 /** @jsxImportSource jsx-htmx */
 
-import {BetterPortalUIView} from "@bettercorp/betterportal";
-import {
-  ViewDefinition,
-} from "./index";
+import { BetterPortalUIView } from "@bettercorp/betterportal";
+import { ViewDefinition } from "./index";
+import fs from 'fs';
+import path from "path";
 
 export const Component: BetterPortalUIView<ViewDefinition> = (props) => {
+  const page = props.outboundData.page,
```

Comment:
Don't create extra variables unless you need too - since you can ref props directly.